src/tests/suite/dfs_test.c

```diff
@@ -278,61 +362,83 @@ dfs_test_short_read(void **state)
 	wsgl.sg_nr = 1;
 	wsgl.sg_iovs = &iov;
 
-	rc = dfs_open(dfs, NULL, name, S_IFREG | S_IWUSR | S_IRUSR,
-		      O_RDWR | O_CREAT, 0, chunk_size, NULL, &obj);
-	assert_int_equal(rc, 0);
+	if (arg->myrank == 0) {
+		rc = dfs_open(dfs_mt, NULL, name, S_IFREG | S_IWUSR | S_IRUSR,
```

Comment:
similarly, we leak open handle here.  Will this cause any issue with dfs_unmount?

Reply:
no it won't cause issues really. that's actually the same issue in all the daos test suite that is using the cmocka assertions. not great, but considering this is test code and the leak happens on failure, im not too concerned.